docs/kits.md

```diff
@@ -74,25 +74,29 @@ To specify a compiler, list the path to the compiler in the `cmake-tools-kits.js
 The most common CMake languages are `C` and `CXX`, and CMake Tools has built-in support for finding these. However, any language can be specified:
 
 ```xml
```

Comment:
Would you mind changing the language specifier to `json` while you're in this file?

Reply:
Hi there, 
I already have json selected (it was automatically recognized!). Or what else could I do? 

It's showing me this:
![image](https://user-images.githubusercontent.com/5083147/232449044-a735869e-4a1a-4716-abed-d30b75bf848e.png)


So when I remove the [] the cmake-kit will not be detected. It is always reproducable.

Reply:
What I meant was that language selected for this code block is `xml` and it should be changed to `json`
![image](https://user-images.githubusercontent.com/12818240/232559927-d061520c-e24d-4d9c-9006-5eac94293613.png)

Reply:
Hi Bob,

I still don't get it sorry. I'm in the cmake-tools-kits.json-file and I have chosen below for language mode "JSON" of course, why should I have it set to xml? It doesn't make any sense to me.

???

Kind Regards
Paddy

Reply:
Don't worry about it. I'll fix it.

I'm talking about this file in the PR.  `docs/kits.md` 
You updated the code blocks that are inside the triple backticks.  There is a language selector on the triple backticks that was incorrectly set to xml years ago (not in your PR).  I was just going to have you fix it while you have the file open for editing, but I can update your branch and push the change.